Demo/Common/IntegrationExamples/AdvancedFlowExample.swift

```diff
@@ -0,0 +1,374 @@
+////
+//// Copyright (c) 2023 Adyen N.V.
+////
+//// This file is open source and available under the MIT license. See the LICENSE file for more info.
+////
+//
+
+import AdyenActions
+import AdyenComponents
+import AdyenDropIn
+
+internal final class AdvancedFlowExample: AdvancedFlowExampleProtocol {
```

Comment:
I think the name should include dropin similar to regular one. Like `DropInAdvancedFlowExample` etc

Reply:
Nice point :). I have renamed it here da858bb9a858774fa139005a65bb215f94ed0c10